helpers/lib/cdnify.js

```diff
@@ -56,7 +56,10 @@ module.exports = globals => {
             }
 
             if (protocol === 'webdav:') {
-                return [cdnUrl, 'content', path].join('/');
+                const imgRegex = /(jpg|jpeg|gif|png)$/i;
+                const isImage = imgRegex.test(path);
+                const prefix = isImage ? 'images/stencil/original/content' : 'content'
+                return [cdnUrl, prefix, path].join('/');
```

Comment:
I feel like a logic as below is safer as it doesn't assume the image extension. What is your thought?
```js
if (path.length >= colonIndex + 5 && path[colonIndex+5] == '/') {
  const assetType = path.slice(colonIndex + 1, colonIndex + 5);
  let prefix = 'content'
  if (assetType == 'img/') {
    prefix = 'images/stencil/original/content';
  }
  return [cdnUrl, prefix, path].join('/');
} else {
   return [cdnUrl, 'content', path].join('/');
}
```

Reply:
I'm not aware that imagers need to be in the `img` path like that, if they are then I would agree. But just looking at my stores image manager images, they are not.

Reply:
If we can indeed rely on webdav images always being in that path then yeah that's definitely way better. (And perhaps most importantly, rely on ONLY images being there)

Reply:
There might be more types we might be missing. But we could make sure we cover all the supported in this https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types

Reply:
I was erring on the side of being conservative just given the history here, but yeah I may be overcompensating. Curious as to the opinion of storefront. 

Also realized I need to regex to match the '.', otherwise it'll match a file just named 'myjpg'.